README.md

```diff
@@ -87,7 +87,9 @@ Then run
 pipenv run python generate_print_files.py <path to config csv> <path to output directory> <supplier> <batch ID> --no-gcs
 ```
 
-This will read the generated qid/uac pairs and generate the print and manifest files in the specified supplier directory
+This will read the generated qid/uac pairs and generate the print and manifest files in the specified supplier directory. 
+Make sure to pass in the correct supplier so the files go to the correct location. The suppliers have been put onto the
```

Comment:
Might be handy to specify what the options are for supplier are, given that there are only 2 allowed options and it's case sensitive.  No biggie though.